components/builder-depot/src/server.rs

```diff
@@ -784,7 +791,16 @@ fn upload_package(req: &mut Request) -> IronResult<Response> {
     let temp_name = format!("{}.tmp", Uuid::new_v4());
     let temp_path = parent_path.join(temp_name);
 
-    let mut archive = write_archive(&temp_path, &mut req.body)?;
+    let mut archive = match write_archive(&temp_path, &mut req.body) {
+        Ok(a) => a,
+        Err(_) => {
```

Comment:
Minor - write out a warn! with the original error code?

Reply:
Yeah, good call.